app/controllers/catalog_controller.rb

```diff
@@ -1,6 +1,8 @@
 class CatalogController < SubsitesController
 
   before_action :refresh_catalog_browse_lists_cache, only: [:home, :browse]
+  
+  self.solr_search_params_logic += [:hide_concepts_when_query_blank_filter]
```

Comment:
@elo2112 Should this search params logic be added to the self.solr_search_params_logic in search_params_logic_behavior.rb? That way all the added methods to  self.solr_search_params_logic are all in one place?